frontend/src/i18n/locales/en.yaml

```diff
@@ -318,6 +318,10 @@ manage:
   access:
     authorized-groups: Authorized groups
     authorized-users: Authorized users
+    workflow-active: >
+      Changing the access policy is not possible at this time, since the video is being processed in the background.
+      <br />
+      Please try again later. To try again now, please reload the page.
```

Comment:
The last sentence sounds a bit odd to me. I would probably remove it and then also remove the `<br />` to put everything in one line. But no strong opinion.

With `<br />` removed, I suppose the commit about adjusting the error display can be removed?